extensions/camel/camel-core/runtime/src/main/java/io/quarkus/camel/core/runtime/CamelRuntime.java

```diff
@@ -62,6 +64,16 @@ public void bind(String name, Class<?> type, Object object) {
 
     public void doInit() {
         try {
+            this.properties = new Properties();
+            Config config = ConfigProvider.getConfig();
+            for (String i : config.getPropertyNames()) {
+                try {
+                    properties.put(i, config.getValue(i, String.class));
+                } catch (Exception e) {
+                    //ignore it if expansion fails
```

Comment:
Hmmm, @lburgazzoli pushed a related PR here: https://github.com/quarkusio/quarkus/pull/1404 with a different approach.

I must admit I would prefer not pushing *all* the properties to Camel if we can avoid that.

In any case, you should reenable the Camel ITs as Luca did.

Reply:
I have rebased it on top of Luca's work and just gone with his approach for now